DSharpPlus/Entities/Interaction/Application/DiscordApplicationCommandOptionChoice.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace DSharpPlus.Entities
+{
+    /// <summary>
+    /// Represents a command parameter choice for a <see cref="DiscordApplicationCommandOption"/>.
+    /// </summary>
+    public sealed class DiscordApplicationCommandOptionChoice
+    {
+        /// <summary>
+        /// Gets the name of this choice parameter. 
+        /// </summary>
+        [JsonProperty("name")]
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Gets the value of this choice parameter. This will either be a type of <see cref="int"/> or <see cref="string"/>.
```

Comment:
This should be `long`, not `int`

Reply:
The docs [here](https://discord.com/developers/docs/interactions/slash-commands#applicationcommandoptionchoice) specifically state integer or string. With that said, would it be better to match the docs or have a larger value?

Reply:
while the docs do say integer, users can pass in `long` values larger than the integer limit that would cause an error while casting as an int.

Reply:
If values that are larger than an `int` cause errors when the data is sent to discord, then this should stay as an `int`. I'll do some testing to see if that is the case.

Reply:
I've already tested it, passing in larger values doesn't cause any error.